Where_Is_My_Wife/Assets/_Main/Scripts/Stage/LavaProyectil.cs

```diff
@@ -0,0 +1,16 @@
+using UnityEngine;
+
+public class LavaProyectil : MonoBehaviour
+{
+    public float speed = 5f;          
+    public float tiempoDeVida = 8f;   
```

Comment:
Use english and not publc fields, change to `[SerializeField] private` in all fields in all scripts